README.en.md

```
@@ -0,0 +1,170 @@
+
+🇩🇪 [Deutsch](README.md) | 🇬🇧 English
+
+This repository provides a comprehensive collection of training materials, sample projects, and implementations for **valve control** using **Eclipse 4diac™**. The focus is on mobile automation, in particular the integration of **ISOBUS (ISO 11783)** technologies.
+
+It is ideally suited for training purposes, master craftsman courses, and developers who want to get into IEC 61131-3 / IEC 61499 programming with distributed as well as local control systems.
+
+logiBUS® <https://www.logibus.tech/>
+
+## 📋 Table of Contents
+
+- [About the Project](#about-the-project)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)